### PR TITLE
feat(*): Freeze code references

### PR DESCRIPTION
**Pull request by qbzzt**
Change the links to the monorepo from links to the develop branch to links to a specific commit. This way, when something changes in the codes, we aren't pointing to the wrong line.

Closing: DEVRL-930

### CODE REVIEW

File: src/docs/developers/bridge/standard-bridge.md [hunk 1]
```diff
@@ -89,21 +89,21 @@ There are two ways to check if a token can use the standard bridge:
...
+ERC20 withdrawals can be triggered via the `withdraw` or `withdrawTo` functions on the [`L2StandardBridge`](https://github.com/ethereum-optimism/optimism/blob/65ec61dde94ffa93342728d324fecf474d228e1f/packages/contracts-bedrock/contracts/L2/L2StandardBridge.sol).
 If you'd like to see this contracts in action, you should check out the [L1 ⇔ L2 deposit-and-withdraw example](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/cross-dom-bridge-erc20).
 
 ### Withdrawing ETH
 
 Unlike on L1, we do not have a separate function on L2 for withdrawing ETH.
-Instead, you can use the `withdraw` or `withdrawTo` functions on the [`L2StandardBridge`](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts/contracts/L2/messaging/L2StandardBridge.sol) and use the address `0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000` as the L2 token address.
+Instead, you can use the `withdraw` or `withdrawTo` functions on the [`L2StandardBridge`](https://linear.app/optimism/issue/DEVRL-984/techical-specifications-page) and use the address `0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000` as the L2 token address.
```
Comment (sbvegan) [inline src/docs/developers/bridge/standard-bridge.md:106]: https://github.com/ethereum-optimism/optimism/blob/65ec61dde94ffa93342728d324fecf474d228e1f/packages/contracts-bedrock/contracts/L2/L2StandardBridge.sol

File: src/docs/developers/build/run-a-node.md [hunk 2]
```diff
@@ -28,11 +28,11 @@ We recommend at least 16 GB RAM, and an SSD drive with at least 500 GB free (for
 ### Source of synchronization
 
 
-[The `op-geth` component](../bedrock/explainer.md#execution-client) synchronizes from both other Optimism nodes (https://github.com/ethereum-optimism/optimism/blob/develop/specs/exec-engine.md#happy-path-sync), meaning L2, [and L1 Ethereum](https://github.com/ethereum-optimism/optimism/blob/develop/specs/exec-engine.md#worst-case-sync) if necessar.
+[The `op-geth` component](../bedrock/explainer.md#execution-client) synchronizes from both [other Optimism nodes, meaning L2](https://github.com/ethereum-optimism/optimism/blob/65ec61dde94ffa93342728d324fecf474d228e1f/specs/exec-engine.md#happy-path-sync), and [L1 Ethereum](https://github.com/ethereum-optimism/optimism/blob/65ec61dde94ffa93342728d324fecf474d228e1f/specs/exec-engine.md#worst-case-sync) if necessar.
```
Comment (sbvegan) [inline src/docs/developers/build/run-a-node.md:31]: typo at the end: necessar --> necessary

File: src/docs/developers/bridge/standard-bridge.md (hunk 1 above)
Comment (qbzzt) [reply src/docs/developers/bridge/standard-bridge.md]: Thank you, fixed.

File: src/docs/developers/build/run-a-node.md (hunk 2 above)
Comment (qbzzt) [reply src/docs/developers/build/run-a-node.md]: Thank you, fixed.

**Review by sbvegan — APPROVED** [review]

**Review by OPMattie — APPROVED** [review]

**Pull request merged**